### PR TITLE
FU-13 authentication-using-Unity-QA-Part-2

### PR DESCRIPTION
**Pull request by bihu1**

### CODE REVIEW

File: furms-web-ui/src/main/java/io/imunity/furms/ui/config/BaseRedirectController.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+package io.imunity.furms.ui.config;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import static io.imunity.furms.ui.constant.LoginFlowConst.LOGIN_URL;
+
+@Controller
+public class BaseRedirectController {
+	@RequestMapping("/")
```
Comment (rkrysinski) [inline furms-web-ui/src/main/java/io/imunity/furms/ui/config/BaseRedirectController.java:15]: @bihu1 question: what will happen if I enter https://localhost:3443/?showSignInOptions.
Will the showSignInOptions parameter be honored?

File: furms-web-ui/src/main/java/io/imunity/furms/ui/config/TokenRevoker.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.net.URI;
+
+import static io.imunity.furms.ui.constant.LoginFlowConst.LOGOUT_SUCCESS_URL;
+import static org.springframework.http.RequestEntity.post;
+
+@Component
+public class TokenRevoker implements LogoutSuccessHandler {
```
Comment (rkrysinski) [inline furms-web-ui/src/main/java/io/imunity/furms/ui/config/TokenRevoker.java:29]: @bihu1 probably can be package private along with c-tor.

File: furms-web-ui/src/main/java/io/imunity/furms/ui/config/TokenRevoker.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+		revokeToken(accessToken, clientId);
+		httpServletResponse.sendRedirect(LOGOUT_SUCCESS_URL);
+	}
+
+	private void revokeToken(String accessToken, String clientId) {
+		URI uriWithParams = UriComponentsBuilder.fromUriString(uri)
+			.queryParam("token", accessToken)
+			.queryParam("client_id", clientId)
+			.queryParam("token_type_hint", "access_token")
```
Comment (rkrysinski) [inline furms-web-ui/src/main/java/io/imunity/furms/ui/config/TokenRevoker.java:57]: @golbi do we also need param "logout" with "true" value here?

File: furms-web-ui/src/main/java/io/imunity/furms/ui/config/BaseRedirectController.java (hunk 1 above)
Comment (bihu1) [reply furms-web-ui/src/main/java/io/imunity/furms/ui/config/BaseRedirectController.java:15]: This controller is not available if you are not log in. Spring security has priority.

**Pull request merged**